app/helpers/number_helper.rb

```diff
@@ -0,0 +1,82 @@
+module NumberHelper
+  def number_to_human_size(number, *args)
+    options = args.extract_options!.reverse_merge(:significant => false, :precision => 1)
+    NumberHelper.handling_negatives(number) do |number|
+      super(number, options)
```

Comment:
[ActionView's NumberHelper](https://github.com/rails/rails/blob/master/actionview/lib/action_view/helpers/number_helper.rb) mixes in to ApplicationHelper, this is included, does it's thing, and calls to the original `number_to_human_size`.